content/en/docs/setup/additional-setup/external-controlplane/snips.sh

```diff
@@ -93,15 +93,28 @@ spec:
     pilot:
       enabled: true
       k8s:
-        volumes:
-        - name: config-volume
-          volumeSource:
-            configMap:
-              localObjectReference:
+        overlays:
+        - kind: Deployment
+          name: istiod
+          patches:
+          - path: spec.template.spec.volumes[100]
```

Comment:
Don't we use [-1] to mean append to end of list? Like in step 3 here: https://istio.io/latest/docs/tasks/traffic-management/egress/wildcard-egress-hosts/#setup-egress-gateway-with-sni-proxy

Reply:
I guess either must work? I copy+pasted from somewhere else.. there is a PR to add support for `volumes` directly so maybe we won't need it for long